data/window.go

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (C) 2023 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package data
```

Comment:
Intellectual exercise, but weren't there libraries we could simply drop in? (Prometheus-esque?)

Reply:
Yes, but your deployment will also require 2 additional containers (prometheus and grafana), this requires additional devops work and costs for hosting.

This app is currently only for the foundation to track production network health. This is the quick and dirty solution for visualising something.